Add entware paths to shell
Index: hyperdiver/rootfs/etc/profile
===================================================================
--- hyperdiver.orig/rootfs/etc/profile
+++ hyperdiver/rootfs/etc/profile
@@ -10,14 +10,17 @@ set_path_before()
 PATH="/usr/bin:/usr/sbin:/bin:/sbin:/mnt/bin"
 set_path_before /usr/local/sbin
 set_path_before /usr/local/bin
+set_path_before /opt/bin
+set_path_before /opt/sbin
 
-LD_LIBRARY_PATH="/usr/local/lib:/usr/lib:/mnt/lib"
+LD_LIBRARY_PATH="/opt/lib:/usr/local/lib:/usr/lib:/mnt/lib"
 
 export PATH
 export LD_LIBRARY_PATH
 
 # ANSI COLORS
 NORMAL="[0;39m"
 RED="[1;31m"
 GREEN="[1;32m"
